sdks/python/apache_beam/runners/interactive/examples/Interactive Beam Running on Flink.ipynb

```diff
@@ -72,66 +74,79 @@
        "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"no\"?>\n",
        "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
        " \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
-       "<!-- Generated by graphviz version 2.38.0 (20140413.2041)\n",
+       "<!-- Generated by graphviz version 2.40.1 (20161225.0304)\n",
```

Comment:
Feels like this generated code shouldn't be checked in...it's a lot of churn.

Reply:
The generated code are dot graphs in the notebook output areas.
This commit renders the new pipeline graph we've developed in the notebook output area, so it contains a lot of changes.
Originally:
![1](https://user-images.githubusercontent.com/4423149/70560901-4ce1a500-1b3e-11ea-8236-2a8446095c0f.png)
Now:
![2](https://user-images.githubusercontent.com/4423149/70560907-51a65900-1b3e-11ea-9316-ae895b1d789e.png)

Reply:
Yes, but should we be checking in auto-generated stuff at all? We should clear the notebook before check in (and the tests should of course run it to see if it still works.) There are lots of irrelevant change here, e.g. you happened to use Python 3.7.5 instead of 2.7.13 this time, that will introduce a lot of churn (e.g. turning a 10 line PR into a several-hundred line PR).

Reply:
Got it, I agree. I'll clean up the `ipynb` files and we shouldn't check in `ipynb` files with outputs.